src/passkit.cs

```diff
@@ -1316,4 +1372,379 @@ interface PKSecureElementPass {
 		[NullAllowed, Export ("pairedTerminalIdentifier")]
 		string PairedTerminalIdentifier { get; }
 	}
+
+	[NoWatch, NoTV]
+	[iOS (14,0)]
```

Comment:
No `[Mac (...)]` attribute?

Reply:
unsure, macOS-side needs to be reviewed (reason for being a draft)